cluster-state-service/swagger/v1/swagger.json

```diff
@@ -248,8 +264,30 @@
     }
   },
   "definitions": {
+    "Metadata": {
+      "type": "object",
+      "required": [
+        "entityVersion"
+      ],
+      "properties": {
+        "entityVersion": {
+          "type": "string"
+        }
+      }
+    },
     "ContainerInstance": {
       "type": "object",
+      "properties": {
+        "metadata": {
+          "$ref": "#/definitions/Metadata"
```

Comment:
Do you think we should keep the metadata object of the container instance independent of that of task? It might be a useful thing to do in case we want to add different metadata information to the two of them. Might be an overkill right niw, not sure!